chia/data_layer/data_store.py

```diff
@@ -783,7 +783,8 @@ async def delete(
                 node = await self.get_node_by_key(key=key, tree_id=tree_id, lock=False)
             else:
                 if bytes(key) not in hint_keys_values:
-                    raise Exception(f"Key not found: {key.hex()}")
+                    log.debug(f"Request to delete an unknown key ignored: {key.hex()}")
+                    return
```

Comment:
For now I think it doesn't matter here but note that changing from raising an exception to returning results in the db context manager committing the transaction instead of rolling it back.  This might be one of the cases where it would be nice to decouple that and be able to tell the transaction to roll back and then we return.

@arvidn, this isn't actually using the v2 wrapper at this point, but the idea is the same.  Our intent here is to _not_ have changed anything and it would be nice to say that to the db wrapper explicitly rather than just being really careful to have not put any db write code ahead of this.